solidity/dashboard/src/pages/TBTCApplicationPage.jsx

```diff
@@ -139,20 +139,47 @@ const TBTCApplicationPage = () => {
           title: "Success",
           content: "Authorization transaction successfully completed",
         })
+        setTimeout(() => onSuccessCallback(contractName, operatorAddress), 5000)
+      } catch (error) {
+        showMessage({
+          type: messageType.ERROR,
+          title: "Authorization action has failed ",
+          content: error.message,
+        })
+        throw error
+      }
+    },
+    [showMessage, web3Context, onSuccessCallback]
+  )
+
+  const deauthorizeTBTCSystem = useCallback(
+    async (data, transactionHashCallback) => {
+      const { operatorAddress } = data
+      try {
+        await tbtcAuthorizationService.deauthorizeTBTCSystem(
+          web3Context,
+          operatorAddress,
+          transactionHashCallback
+        )
+        showMessage({
+          type: messageType.SUCCESS,
+          title: "Success",
+          content: "Deauthorization transaction successfully completed",
+        })
         setTimeout(
-          () => onAuthorizationSuccessCallback(contractName, operatorAddress),
+          () => onSuccessCallback("TBTCSystem", operatorAddress, true),
```

Comment:
also here, depending on the comment regarding `isDeathorization` we might want to pass `false` here